otx/algorithms/action/adapters/mmaction/data/cls_dataset.py

```diff
@@ -109,6 +109,7 @@ def __init__(
     ):
         self.otx_dataset = otx_dataset
         self.labels = labels
+        self.CLASSES = list(label.name for label in labels)
```

Comment:
```suggestion
        self.CLASSES = [label.name for label in labels]
```
It doen't matter, but this is more recommended way :)